tests/src/FunctionalJavascript/TranslationTestBase.php

```diff
@@ -53,10 +58,16 @@ protected function setUp() {
     $this->container->get('content_translation.manager')->setEnabled('node', 'page', TRUE);
     $this->container->get('router.builder')->rebuild();
 
+    $this->drupalPlaceBlock('page_title_block', ['region' => 'content']);
+    $this->drupalPlaceBlock('local_tasks_block', ['region' => 'content']);
+    $this->drupalPlaceBlock('local_actions_block', ['region' => 'content']);
+
     /** @var \Drupal\user\RoleInterface $role */
     $role = $this->entityTypeManager->getStorage('user_role')->load('oe_translator');
     $permissions = $role->getPermissions();
     $permissions[] = 'administer menu';
+    $permissions[] = 'edit any page content';
+    $permissions[] = 'administer site configuration';
```

Comment:
Well, we cannot add this permission that essentially allows them to do anything. It's no longer a translator then. You need to keep this user a regular translator, which may create some content, fine, but not do everything. Make another user where you need the proper role for creating provider entities.